### ISSUE TITLE
Fix unit tests in logs microservice

### ISSUE THREAD

**Issue by suryajak** [tmobile/jazz#625]
### Description

A few unit tests are broken in the logs microservice (referenced in this forked repo: https://github.com/suryajak/jazz/commit/6b5afc0abd3ff168170c3c742f6ebe4a21833360) which will need to be fixed once this is merged to master.

### Steps to Reproduce

1. Uncomment the tests in the referenced commit and run the unit tests

**Expected behavior:** [What you expect to happen]
Tests should pass

**Actual behavior:** [What actually happens]
Tests fail 

**Reproduces how often:** [What percentage of the time does it reproduce?]
100%

**Comment by davidsims9t** [comment]
Where is `npm test` defined? I don't see any package.json file.

**Comment by davidsims9t** [comment]
I created a PR for this issue: https://github.com/suryajak/jazz/pull/2

**Comment by suryajak** [comment]
Thanks @davidsims9t. The description is misleading in the sense that it is using a commit id to point out the issue but the commit id is from a forked repo. Can you raise a PR fixing this issue in 
https://github.com/tmobile/jazz/blob/master/core/jazz_logs/test/test.js?

Specifically the following unit tests needs to be uncommented:

1. https://github.com/tmobile/jazz/blob/master/core/jazz_logs/test/test.js#L255 - https://github.com/tmobile/jazz/blob/master/core/jazz_logs/test/test.js#L264 
2. https://github.com/tmobile/jazz/blob/master/core/jazz_logs/test/test.js#L271 - https://github.com/tmobile/jazz/blob/master/core/jazz_logs/test/test.js#L297
3. https://github.com/tmobile/jazz/blob/master/core/jazz_logs/test/test.js#L341 - https://github.com/tmobile/jazz/blob/master/core/jazz_logs/test/test.js#L373

Let me know if that makes sense.

**Comment by davidsims9t** [comment]
@suryajak Yes I opened a PR against this repo. Also I noticed you didn't mention the test on lines 303 to 334. Should I ignore this test?
